Bottom: 8f4ed29eb8c6f64110242d09b985eaad5af0acc3
Top:    db822a49c3c85e25f4241513c8eabe6a34441413
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 16:45:48 -0500

Remove variables from credential file paths.

Unnecessary, and was causing issues with recursively created
directories of credentials.


---
diff --git a/ansible-playbook/group_vars/all/credentials.yml b/ansible-playbook/group_vars/all/credentials.yml
index 02a1cad..6e11e08 100644
--- a/ansible-playbook/group_vars/all/credentials.yml
+++ b/ansible-playbook/group_vars/all/credentials.yml
@@ -1,22 +1,24 @@
 ---
 # passwords
-ldap_admin_password:                 '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.ldap.hostname + "/cn=admin,dc=novatech length=20") }}'
-ldap_proxyagent_password:            '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.ldap.hostname + "/cn=proxyagent,dc=novatech length=20") }}'
-wiki_db_root_password:               '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.wiki.hostname + "/db/root length=20") }}'
-wiki_db_password:                    '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.wiki.hostname + "/db/" + wiki_db_user + " length=20") }}'
-wiki_email_password:                 '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.wiki.hostname + "/email/wiki@novatech-llc.com length=20") }}'
-gitlab_db_password:                  '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.git.hostname + "/db/" + gitlab_db_user + " length=20") }}'
-gitlab_root_password:                '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.git.hostname + "/users/root length=20") }}'
-gitlab_email_password:               '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.git.hostname + "/email/gitlab@novatech-llc.com length=20") }}'
-gitlab_secrets_db_key_base:          '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.git.hostname + "/key/GITLAB_SECRETS_DB_KEY_BASE length=40") }}'
-mantisbt_db_root_password:           '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.mantisbt.hostname + "/db/root length=20") }}'
-mantisbt_db_password:                '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.mantisbt.hostname + "/db/" + mantisbt_db_user + " length=20") }}'
-mantisbt_email_password:             '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/" + container_addr_map.mantisbt.hostname + "/email/mantis@novatech-llc.com length=20") }}'
-buildsystem_ssh_prod_user_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/ssh/prod length=20") }}'
-buildsystem_ssh_root_user_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/ssh/root length=20") }}'
-buildsystem_database_root_password:  '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/database/root length=20") }}'
-buildsystem_database_build_password: '{{ lookup("password", inventory_dir + "/credentials/buildsystem.novatech-llc.com/database/readuser") }}'
-svn_username:                        '{{ lookup("file",     inventory_dir + "/personal_credentials/svn/username") }}'
-svn_password:                        '{{ lookup("file",     inventory_dir + "/personal_credentials/svn/password") }}'
-buildbot_worker_ntel_password:       '{{ lookup("password", inventory_dir + "/credentials/buildbot.novatech-llc.com/worker_ntel length=20") }}'
-buildbot_worker_ptxdist_password:    '{{ lookup("password", inventory_dir + "/credentials/buildbot.novatech-llc.com/worker_ntel length=20") }}'
+ldap_admin_password:                 '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=admin,dc=novatech length=20") }}'
+ldap_proxyagent_password:            '{{ lookup("password", playbook_dir + "/credentials/ldap/cn=proxyagent,dc=novatech length=20") }}'
+wiki_db_root_password:               '{{ lookup("password", playbook_dir + "/credentials/wiki/db/root length=20") }}'
+wiki_db_password:                    '{{ lookup("password", playbook_dir + "/credentials/wiki/db/user length=20") }}'
+wiki_email_password:                 '{{ lookup("password", playbook_dir + "/credentials/wiki/email/wiki@novatech-llc.com length=20") }}'
+gitlab_db_password:                  '{{ lookup("password", playbook_dir + "/credentials/git/db/user length=20") }}'
+gitlab_root_password:                '{{ lookup("password", playbook_dir + "/credentials/git/users/root length=20") }}'
+gitlab_email_password:               '{{ lookup("password", playbook_dir + "/credentials/git/email/gitlab@novatech-llc.com length=20") }}'
+gitlab_secrets_db_key_base:          '{{ lookup("password", playbook_dir + "/credentials/git/key/GITLAB_SECRETS_DB_KEY_BASE length=40") }}'
+mantisbt_db_root_password:           '{{ lookup("password", playbook_dir + "/credentials/mantis/db/root length=20") }}'
+mantisbt_db_password:                '{{ lookup("password", playbook_dir + "/credentials/mantis/db/user length=20") }}'
+mantisbt_email_password:             '{{ lookup("password", playbook_dir + "/credentials/mantis/email/mantis@novatech-llc.com length=20") }}'
+buildsystem_ssh_prod_user_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/ssh/prod length=20") }}'
+buildsystem_ssh_root_user_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/ssh/root length=20") }}'
+buildsystem_database_root_password:  '{{ lookup("password", playbook_dir + "/credentials/buildsystem/database/root length=20") }}'
+buildsystem_database_build_password: '{{ lookup("password", playbook_dir + "/credentials/buildsystem/database/readuser") }}'
+svn_password:                        '{{ lookup("password", playbook_dir + "/credentials/svn/ansible length=20") }}'
+buildbot_admin_password:             '{{ lookup("password", playbook_dir + "/credentials/buildbot/admin length=20") }}'
+buildbot_worker_ntel_password:       '{{ lookup("password", playbook_dir + "/credentials/buildbot/worker_ntel length=20") }}'
+buildbot_worker_ptxdist_password:    '{{ lookup("password", playbook_dir + "/credentials/buildbot/worker_ptxdist length=20") }}'
+
+svn_username:                        'ansible'
diff --git a/ansible-playbook/group_vars/bacula/credentials.yml b/ansible-playbook/group_vars/bacula/credentials.yml
index e0a390a..4badf5a 100644
--- a/ansible-playbook/group_vars/bacula/credentials.yml
+++ b/ansible-playbook/group_vars/bacula/credentials.yml
@@ -1,6 +1,6 @@
 ---
 # passwords
-mariadb_root_password:               '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/mariadb/root length=20") }}'
-bacula_director_password:            '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/bacula/director length=20") }}'
-bacula_monitor_password:             '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/bacula/monitor length=20") }}'
-mariadb_bacula_password:             '{{ lookup("password", inventory_dir + "/credentials/" + inventory_hostname + "/bacula/mariadb length=20") }}'
+mariadb_root_password:               '{{ lookup("password", playbook_dir + "/credentials/mariadb/root length=20") }}'
+bacula_director_password:            '{{ lookup("password", playbook_dir + "/credentials/bacula/director length=20") }}'
+bacula_monitor_password:             '{{ lookup("password", playbook_dir + "/credentials/bacula/monitor length=20") }}'
+mariadb_bacula_password:             '{{ lookup("password", playbook_dir + "/credentials/bacula/mariadb length=20") }}'
